### DISCUSSION TITLE
Flutter MVVM Architecture Generator

### DISCUSSION THREAD

**Discussion by gdelgado11** [rrousselGit/riverpod#776] [Q&A] [answered]
Hello I hope you are well.
I wanted to ask you if you didn't think it was a good idea to create a plugin for vscode that allows you to generate MVVM Architecture with Riverpod, really that would be great for those of us who are just starting out, since it gives us structure and order when working, not to mention that it allows us to accelerate the times. developmental.
I know there are Snipets but if it could be complemented with the project generator it would be great.
I have seen that with Artemis I can generate the Models and with Frezzed something is also generated.
I thank you in advance for your time and the great contribution you have made. 
Regards,
GD

**Comment by rrousselGit** [comment, marked as answer]
Sorry for the late reply

I'm not against it, but I don't have the time to do it personally in the near future. It's not a priority for me.
